source/normalize-arguments.ts

```diff
@@ -62,27 +67,28 @@ export const preNormalizeArguments = (options: any, defaults?: any) => {
 		methods: new Set(),
 		statusCodes: new Set(),
 		errorCodes: new Set(),
-		maxRetryAfter: undefined
+		maxRetryAfter: 0
```

Comment:
:+1:

Reply:
need to update readme also

Reply:
actually this needs to be greater than 0 because by default it won't retry on `Retry-After`. Something like `30000`.